_pages/resources.md

```diff
@@ -10,8 +11,7 @@ redirect_from:
     - /ta_titleiii.html
 ---
 
-{% include ta-link-back.html %}
-
+{% include common-alert.html type="info" text="The complete inventory of our ADA Resources is coming soon to this page. In the meantime, you can access them on our former Technical Assistance Materials page." %}
```

Comment:
According to https://github.com/usdoj-crt/beta-ada/pull/471#issuecomment-1380691616, this should have a hyperlink

Reply:
added link back